tasks/hyrax-metadata-updates/index.js

```diff
@@ -310,6 +310,12 @@ const updateGranule = (config) => async (granule) => {
   const metadataFile = granule.files.find(isCMRFile);
   // If there is no metadata file, error out.
   if (metadataFile === undefined) {
+    if (config.allowNoMetadata) {
```

Comment:
In the ticket (https://bugs.earthdata.nasa.gov/browse/CUMULUS-2953) they're asking for a flag called `skipMetaCheck`. That makes me wonder if there's a scenario where the file actually *does* have metadata but we want to explicitly skip it anyway. Is that a possible scenario?

I think what you have here is probably correct but the naming of the variable in the user request just gave me pause.

Reply:
Confirmed with @amarouane-ABDELHAK that this is not a use case.